spec/models/miq_expression_spec.rb

```diff
@@ -1631,4 +1631,146 @@
       end
     end
   end
+
+  describe "#sql_supports_atom?" do
+    context "expression key is 'CONTAINS'" do
+      context "operations with 'tag'" do
+        it "returns true if format of field is /\w+\.\w+/ - interpreted as tag of the main model" do
+          # example of expression:
+          # expression = {"CONTAINS" => {"tag" => "VmInfra.managed-operations", "value" => "analysis_failed"}}
+          expression = {"CONTAINS" => {"tag" => "HelloWorld.aaa"}}
```

Comment:
question is: 
is is better to highlight what is important and give hints for refactoring but tie spec to implementation
OR
 put not used data/parameter in test call ?